training/nodejs/js/index.js

```diff
@@ -1,9 +1,9 @@
 const https = require('https') 
 
-const FEED_URl = "https://api.nasa.gov/neo/rest/v1/feed"
+const API_KEY = process.env.NASA_API_KEY
+const FEED_URl = process.env.NASA_API_FEED_URL
 const START_DATE = "2024-02-26"
```

Comment:
it's a bad way to store dates as hardcoded string,  probably you will want to take a look at moment library

Reply:
Fixed